Source/WebCore/platform/graphics/cg/ImageBufferUtilitiesCG.cpp

```diff
@@ -96,9 +96,21 @@ ALLOW_DEPRECATED_DECLARATIONS_BEGIN
 ALLOW_DEPRECATED_DECLARATIONS_END
 }
 
-static bool encode(CGImageRef image, CFStringRef destinationUTI, std::optional<double> quality, const ScopedLambda<PutBytesCallback>& function)
+bool platformMIMETypeIsJPEG(const String& mimeType)
```

Comment:
I do not think we need different platform implementations for this function. I think we can just use one function named MIMETypeRegistry::isJPEGMIMEType() for example.

Reply:
Moved to MIMETypeRegistry.cpp as isJPEGMIMEType.